app/src/main/java/com/android/solvit/seeker/ui/navigation/BottomNavigationMenu.kt

```diff
@@ -90,30 +91,43 @@ fun SeekerBottomNavigationMenu(
               val filteredTabList = tabList.filter { it.route != Route.CREATE_REQUEST }
               filteredTabList.forEachIndexed { index, tab ->
                 BottomNavigationItem(
-                    icon = { Icon(tab.icon, contentDescription = null, tint = Color.White) },
+                    icon = {
+                      Icon(
+                          tab.icon,
+                          contentDescription = null,
+                          tint =
+                              if (tab.route == selectedItem) Color(0xFF0099FF)
+                              else Color(0xFFD8D8D8))
+                    },
                     selected = tab.route == selectedItem,
                     onClick = { onTabSelect(tab) },
                     modifier = Modifier.testTag(tab.textId))
               }
             }
-        if (tabList == LIST_TOP_LEVEL_DESTINATION_CUSTOMER) {
-          FloatingActionButton(
-              onClick = { onTabSelect(TopLevelDestinations.CREATE_REQUEST) },
-              modifier =
-                  Modifier.size(70.dp)
-                      .offset(y = (-25).dp)
-                      .offset(x = (142).dp)
-                      .align(Alignment.TopCenter)
-                      .testTag(TopLevelDestinations.CREATE_REQUEST.toString()),
-              shape = CircleShape,
-              containerColor = Color(0xFF0099FF)) {
-                Icon(
-                    Icons.Outlined.Add,
-                    contentDescription = "Add",
-                    tint = Color.White,
-                    modifier = Modifier.size(30.dp))
-              }
-        }
+
+        FloatingActionButton(
+            onClick = {
+              onTabSelect(
+                  if (tabList == LIST_TOP_LEVEL_DESTINATION_CUSTOMER)
+                      TopLevelDestinations.CREATE_REQUEST
+                  else TopLevelDestinations.MYJOBS)
+            },
+            modifier =
+                Modifier.size(height * 0.85f)
+                    .offset(y = (-25).dp)
+                    .offset(x = width * 0.5f - 53.dp)
+                    .align(Alignment.TopCenter)
+                    .testTag(TopLevelDestinations.CREATE_REQUEST.toString()),
+            shape = CircleShape,
+            containerColor = Color(0xFF0099FF)) {
+              Icon(
+                  if (tabList == LIST_TOP_LEVEL_DESTINATION_CUSTOMER) Icons.Outlined.Add
+                  else Icons.Outlined.CheckCircle,
+                  contentDescription =
+                      if (tabList == LIST_TOP_LEVEL_DESTINATION_CUSTOMER) "Add" else "Myjobs",
```

Comment:
Nit: The content description does not seem to be consistent with the textId for both Screen and the My Jobs top level destination. It could be changed to "My Jobs"